nva/nvafakebios.c

```diff
@@ -80,8 +80,12 @@ int vbios_upload_pramin(int cnum, uint8_t *vbios, int length)
 			cnum, nva_cards[cnum]->chipset.chipset);
 
 	if (nva_cards[cnum]->chipset.card_type >= 0x50) {
-		uint64_t vbios_vram = (uint64_t)(nva_rd32(cnum, 0x619f04) & ~0xff) << 8;
-
+		if (vbios_vram)
+			/* If vbios vram addr was already chosen, save the addr into 0x619f04
+			 * 0x1 tells it's vram and 0x8 enables it */
+			nva_wr32(cnum, 0x619f04, ((vbios_vram >> 8) & ~0xff) | 0x09);
```

Comment:
I don't think we actually require to write in there. Mind testing if skipping it entirely and only the the write to 0x1700 further below is actually enough?

Reply:
doesn't work without this on gf119